modules/virt-preventing-workload-updates-during-control-plane-only-update.adoc

```diff
@@ -8,6 +8,28 @@
 
 When you update from one Extended Update Support (EUS) version to the next, you must manually disable automatic workload updates to prevent {VirtProductName} from migrating or evicting workloads during the update process.
 
+[IMPORTANT]
+====
+In {product-title} 4.16, the underlying {op-system-first} upgraded to version 9.4 of {op-system-base-full}. To operate correctly, all `virt-launcher` pods in the cluster need to use the same version of {op-system-base}.
+
+After upgrading to {product-title} 4.16 from an earlier version, re-enable workload updates in {VirtProductName} to allow `virt-launcher` pods to update. Before upgrading to the next {product-title} version, verify that all VMIs use up-to-date workloads:
+
+[source,terminal]
+----
+$ oc get kv kubevirt-kubevirt-hyperconverged -o json -n openshift-cnv | jq .status.outdatedVirtualMachineInstanceWorkloads
+0
+----
+
+If the previous command returns a value larger than 0, list all VMIs with outdated `virt-launcher` pods and live migrate them to a new version manually:
+
+[source,terminal]
+----
+$ oc get vmi -l kubevirt.io/outdatedLauncherImage --all-namespaces
+----
+
+For the list of supported {product-title} releases and the {op-system-base} versions they use, see link:https://access.redhat.com/articles/6907891[RHEL Versions Utilized by RHEL CoreOS and OCP].
```

Comment:
```suggestion
For the list of supported {product-title} releases and the {op-system-base} versions they use, see link:https://access.redhat.com/articles/6907891[RHEL Versions Utilized by {op-system-bas} CoreOS and {product-title}].
```

Reply:
Best to stick with OpenShift Container PLatform as OCP is kind of outlawed in our docs. I understand the title in the source page does state "OCP".

See https://redhat-documentation.github.io/supplementary-style-guide/#red-hat-openshift-container-platform

Reply:
Fixed, I know about the style guide but feel icky about faking the titles of linked articles. That said, if that is the preferred way, I used a different combination of attributes to use abbreviated versions of products that have already been introduced.